src/component/tooltip/TooltipView.ts

```diff
@@ -394,6 +394,7 @@ class TooltipView extends ComponentView {
         api: ExtensionAPI,
         payload: HideTipPayload
     ) {
+        clearTimeout(this._showTimout);
```

Comment:
I think we should make a new private function called `_clearTimeout` and in this function, check `this._showTimeout != null` before calling `clearTimeout` and set `this._showTimout` to be null after that.

Both `manuallyHideTip` and `_showOrMove` should call this new function instead of `clearTimeout(this._showTimout);`.

Reply:
Thank you for your advice,your suggestion is very good.
But it is unnecessary because we need to restart the timer after stopping it, not clear variable '_ Showtimout'.

void WindowOrWorkerGlobalScope::clearTimeout(EventTarget& event_target,int timeout_id){}  Function of wrong type will not be executed.Therefore, no judgment is required.

The biggest reason is that I am not familiar with Typescript, hahaha!!!

Reply:
I just studied Typescript. Adding a private function is not complicated, but I still think it is unnecessary.What about you?